src/cascadia/TerminalConnection/CTerminalHandoff.h

```diff
@@ -0,0 +1,58 @@
+/*++
+Copyright (c) Microsoft Corporation
+Licensed under the MIT license.
+
+Module Name:
+- CTerminalHandoff.h
+
+Abstract:
+- This module receives an incoming request to host a terminal UX
+  for a console mode application already started and attached to a PTY.
+
+Author(s):
+- Michael Niksa (MiNiksa) 31-Aug-2020
+
+--*/
+
+#pragma once
+
+#include "ITerminalHandoff.h"
+
+#if defined(WT_BRANDING_RELEASE)
+#define __CLSID_CTerminalHandoff "E12CFF52-A866-4C77-9A90-F570A7AA2C6B"
+#elif defined(WT_BRANDING_PREVIEW)
+#define __CLSID_CTerminalHandoff "86633F1F-6454-40EC-89CE-DA4EBA977EE2"
+#else
+#define __CLSID_CTerminalHandoff "051F34EE-C1FD-4B19-AF75-9BA54648434C"
+#endif
+
+using namespace Microsoft::WRL;
+
+typedef HRESULT (*NewHandoff)(HANDLE, HANDLE, HANDLE, HANDLE);

```

Comment:
```suggestion
using NewHandoffFunction = HRESULT (*)(HANDLE, HANDLE, HANDLE, HANDLE);
```

I believe this is the correct syntax.